features/lib/laravel.rb

```diff
@@ -21,7 +21,11 @@ def navigate_to(route, attempts = 0)
     end
 
     def fixture
-      ENV.fetch('LARAVEL_FIXTURE', 'laravel56')
+      fixtureName = ENV.fetch('LARAVEL_FIXTURE', 'laravel56')
+      if fixtureName == "laravelrr" || fixtureName == "laravelfp" || fixtureName == "laravelsw"
```

Comment:
Could we replace this with a static array at the top and test it using `include?`

```
OCTANE_FIXTURES = ['laravelrr', 'laravelfp', 'laravelsw']
if OCTANE_FIXTURES.include?(fixtureName) ...
```